components/bus/marshaler_protobuf_test.go

```diff
@@ -0,0 +1,55 @@
+package bus_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/components/bus"
+)
+
+func TestProtobufMarshaler(t *testing.T) {
+	marshaler := bus.ProtobufMarshaler{}
+
+	when, err := ptypes.TimestampProto(time.Now())
+	require.NoError(t, err)
+	eventToMarshal := &TestProtobufEvent{
+		Id:   watermill.NewULID(),
+		When: when,
+	}
+
+	msg, err := marshaler.Marshal(context.Background(), eventToMarshal)
+	require.NoError(t, err)
+
+	eventToUnmarshal := &TestProtobufEvent{}
+	err = marshaler.Unmarshal(msg, eventToUnmarshal)
+	require.NoError(t, err)
+
+	assert.EqualValues(t, eventToMarshal.String(), eventToUnmarshal.String())
+	assert.Equal(t, msg.Metadata.Get("name"), "cqrs_test.TestProtobufEvent")
```

Comment:
Probably a copypaste, the package name shouldn't be `cqrs_test`.
Or, according to the PR description, there shouldn't be a package name at all